internal/generate/clusterserviceversion/bases/markers.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bases
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/markbates/inflect"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/operator-framework/operator-sdk/internal/generate/olm-catalog/descriptor"
+)
+
+func updateDescriptionsForGVKs(csv *operatorsv1alpha1.ClusterServiceVersion, apisDir string,
```

Comment:
This was copied verbatim from [here](https://github.com/operator-framework/operator-sdk/blob/v0.17.0/internal/generate/olm-catalog/csv_updaters.go#L289).